### ISSUE TITLE
Page rendering issues on Brave/Chrome (WINDOWS 7)

### ISSUE THREAD

**Issue by krimnassim** [brave/brave-browser#8795]
i used brave for a year with no problem my issue with brave on windows 7 a lot of site not working i get site not secure and sites load like this image https://imgur.com/a/os96ItM  and MEGA.NZ  is loading like this https://imgur.com/a/aRI5H4u but when i switch to fire fox no problem all working i tested brave on windows 10 pro working great i only have this issues on windows 7 i really need brave on w7 thanx.
![iss](https://user-images.githubusercontent.com/62130284/77212080-2c24d600-6ac3-11ea-92cc-cfbc74452940.PNG)
![Capture](https://user-images.githubusercontent.com/62130284/77212083-31822080-6ac3-11ea-9772-c3a930b5d179.PNG)

**Comment by outforpublic** [comment]
Have you tried:

1. Exiting out of brave and opening up again
2. Clearing all cache & cookies (via brave://settings/clearBrowserData)
3. Checking if any updates are available (via brave://settings/help)
4. Rebooting your PC
5. Uninstalling and Reinstalling brave

**Comment by bsclifton** [comment]
Definitely looks like a cache problem; thanks for the suggestions, @outforpublic - I would also recommend trying those too 😄

**Comment by krimnassim** [comment]
hi, thanx for the help yes i did all that but same problem i even reinstalled windows 7 Twice same issue but when i use firefox no problem all the sits working , i installed windows 10 brave works with no problem this issue happen only when i use brave on windows 7, i didn't have any problem before using brave on w7, this issue started last weak.

**Comment by outforpublic** [comment]
Is this reproducible in Chrome/Chromium?

**Comment by krimnassim** [comment]
yes same issue on chrome and chromium i tested other browsers to, only fire fox is working i dont know why

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
